### PR TITLE
added a delete button

### PR DESCRIPTION
**Pull request by JFleming4**
The delete button isn't right justified because it was causing  alignment issues and I don't want to fight with css right now

### CODE REVIEW

**Comment by JFleming4** [comment]
screen shot 2018-04-05 at 8 40 50 am

File: src/main/java/app/controllers/PresentationController.java [hunk 1]
```diff
@@ -79,6 +73,22 @@ public String updateTimeSlots(@PathVariable Long pid,
                 return "redirect:/facultyMenu";
             }
         }
+        if(!action.toLowerCase().equals("update")) {
+            if(authenticatedUser.getType().equals("Student")) {
+                return "redirect:/project/" + pid + "/presentation";
+            }
+            proj.removeTimeSlot(new Long(action));
```
Comment (DerekStride) [inline src/main/java/app/controllers/PresentationController.java]: Shouldn't action be `delete` and you pass in the timeSlot id via the request param?

Comment (JFleming4) [reply src/main/java/app/controllers/PresentationController.java]: Id have to add a hidden field for the param which seems hacky

**Review by DerekStride — APPROVED** [review]
I do think using an ajax call would have ended up being better but this is fine.

this_is_fine

**Pull request merged**